CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesela V100*1/60 GB/12 CPU",
+	    "latency": 0.683,
+	    "cost": 2.078e-3,

```

Comment:
I am not familiar with Baidu Cloud's pricing. Can you send me a link to the instance price for this machine?

Reply:
https://cloud.baidu.com/doc/GPU/Pricing.html#.E5.8C.85.E5.B9.B4.E5.8C.85.E6.9C.88.E8.AE.A1.E8.B4.B9

It is in Chinese, and its price is 5,399 RMB/ month, so its corresponding price in USD is about 805.82